src/main/java/webserver/App.java

```diff
@@ -5,23 +5,27 @@
 import java.net.ServerSocket;
 import java.net.Socket;
 
-public class App {
+public class App {    
+    private static SocketCreator socketCreator = new SocketCreator();
+    private static SystemUtils system = new SystemUtils();
```

Comment:
This looks like a good example of wrapping an API, which will make it easier to manage dependencies, test code going, and extend code going forward.  I imagine this class probably calls `System.getProperty` right now, but you could (if needed in the future; not now) turn `SystemUtils` into an interface and have the implementation do different things (read from a file, read from the keyboard, or be mocked in a test, etc...).